generators/app/index.js

```diff
@@ -1,13 +1,13 @@
-'use strict';
-var yeoman = require('yeoman-generator');
-var chalk = require('chalk');
-var yosay = require('yosay');
-var randtoken = require('rand-token');
-var _ = require('lodash');
+const generator = require('yeoman-generator');
+const chalk = require('chalk');
+const yosay = require('yosay');
+const randtoken = require('rand-token');
+const { startCase, kebabCase } = require('lodash');
 
-module.exports = yeoman.Base.extend({
-  prompting: function () {
-    var that = this;
+module.exports = class extends generator {  
```

Comment:
`Generator`

Reply:
I changed this.. probably i forgot to do it in this file...